lib/node_modules/@stdlib/lapack/base/dpttrf/docs/repl.txt

```diff
@@ -0,0 +1,108 @@
+
+{{alias}}( N, D, E, info )
+    Computes the `L * D * L ** T` factorization of real symmetric positive
+    definite tridiagonal matrix `A`.
+
+    Indexing is relative to the first index. To introduce an offset, use typed
+    array views.
+
+    Parameters
+    ----------
+
+    N: integer
+        Order of matrix `A`.
+
+    D: Float64Array
+        Diagonal elements of `A`.
+
+    E: Float64Array
+        Subdiagonal elements of `A`.
+
+    info: integer
+        Status code.
+
+    Returns
+    -------
+    info: integer
+        Status code.
```

Comment:
I think this and the equivalent for `ndarray` can be fairly automated, I'll just need the jsdoc or readme description and it will be done.